src/alloc.rs

```diff
@@ -12,4 +12,7 @@ pub fn call() {
 		data.extend_from_slice(&[5u8; 400*1024][..]);
 		data
 	});
-}
\ No newline at end of file
+}
+
+#[no_mangle]
+pub fn deploy() { }
```

Comment:
I wonder why these are needed?

Reply:
`wasm-build` would complain if no `deploy` symbol is found: https://github.com/paritytech/wasm-utils/blob/master/src/build.rs#L110
In the binary, we always set [constructor to true](https://github.com/paritytech/wasm-utils/blob/master/src/build.rs#L108) and build the deploy wrapped version.

Reply:
@sorpaas 

there is a bug in wasm-build 0.3.0 with this constructor=true always

you might try pwasm-utils-cli 0.2.0 (`cargo install pwasm-utils-cli --version 0.2.0`) or wait until @fckt fix it

Reply:
https://github.com/paritytech/wasm-utils/pull/100